README.md

```diff
@@ -332,7 +336,7 @@ bicleaner-train \
           --classifier_type random_forest
 ```
 
-This will train a Random Forest classifier for English-Czech using the corpus corpus.en-cs.train and the probabilistic dictionaries `dict-en-cs.gz` and `dict-cs-en.gz`. 
+This will train a Random Forest classifier for English-Czech using the corpus corpus.en-cs.train, the probabilistic dictionaries `dict-en-cs.gz` and `dict-cs-en.gz`, and the word frequency dictionaries `wordfreqs-en.gz` and `wordfreqs-cs.gz`.
```

Comment:
Please explain  word frequency dictionaries

Reply:
Does this line need to explain word frequency lists if it explained on the requirements? Probabilistic dictionaries are not explained here either.